app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,20 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = Comment.new
```

Comment:
- [ ] I would suggest that you should create the comment in this way ``@comment = @current_user.comments.new``